app/views/favorites/index.html.erb

```diff
@@ -20,7 +20,7 @@
 <h2>Pending Applications</h2>
 
 <section id="pending-applications">
-  <% PetApplication.get_pets.each do |pet| %>
+  <% @allpets.pets_with_applications.each do |pet| %>
```

Comment:
I wanted to reduce use of database queries in views. With the new `@allpets`, we are able to call the class method on it which returns the required array of pets.